Zend/zend_property_hooks.c

```diff
@@ -275,9 +274,8 @@ static void zho_it_rewind(zend_object_iterator *iter)
 	zend_array *properties = Z_ARR(hooked_iter->declared_props);
 	zend_hash_internal_pointer_reset(properties);
 	hooked_iter->dynamic_props_done = false;
-	if (hooked_iter->dynamic_prop_it != (uint32_t) -1) {
-		EG(ht_iterators)[hooked_iter->dynamic_prop_it].pos = zho_num_backed_props(Z_OBJ(iter->data));
-	}
+	ZEND_ASSERT(hooked_iter->dynamic_prop_it != (uint32_t) -1);
```

Comment:
This is no longer assigned `-1` anywhere, so this assert is kinda useless.

Reply:
Oh right